### ISSUE TITLE
iotile-sensorgraph - trigger_streamer statement does not get created

### ISSUE THREAD

**Issue by x24ling** [iotile/coretools#410]
iotile-sensorgraph 0.4.0 was the lastest version that did have this issue.

**Comment by x24ling** [comment]
```
(system input 2 always) => counter 1024 using copy_all_a
(system input 3 always) => counter 1025 using copy_all_a
(system input 5 always) => counter 1026 using copy_all_a
(system input 6 always) => counter 1027 using copy_all_a
(system input 1025 always) => unbuffered 1024 using copy_latest_a
(system input 1026 always) => unbuffered 1025 using copy_latest_a
(counter 1024 when count >= 1) => counter 1028 using copy_latest_a
(constant 1026 always && unbuffered 1024 when value == 8) => constant 1025 using copy_latest_a
(constant 1027 always && unbuffered 1025 when value == 8) => constant 1025 using copy_latest_a
(unbuffered 1024 always) => unbuffered 1026 using copy_latest_a
(counter 1024 when count >= 360) => counter 1031 using copy_latest_a
(counter 1028 when count == 1 && constant 1024 always) => output 15 using call_rpc
(constant 1025 always) => constant 1028 using copy_latest_a
(unbuffered 1026 when count == 1 && constant 1029 always) => unbuffered 1027 using trigger_streamer
(system input 0 always && counter 1031 when count == 1) => system output 0 using copy_latest_a
(counter 1025 always && constant 1028 when value == 1) => counter 1029 using copy_latest_a
(counter 1029 when count >= 1) => counter 1030 using copy_latest_a
(counter 1030 when count == 1 && constant 1030 always) => unbuffered 39 using call_rpc
```

After optimization, the trigger statement line is removed after the RemoveDeadCodePass optimization step
```
(unbuffered 1026 when count == 1 && constant 1029 always) => unbuffered 1027 using trigger_streamer
```

**Comment by x24ling** [comment]
Closed by #412
